concrete/src/Cache/Adapter/ZendCacheDriver.php

```diff
@@ -87,12 +138,14 @@ protected function internalGetItem(&$normalizedKey, &$success = null, &$casToken
      */
     protected function internalSetItem(&$normalizedKey, &$value)
     {
-        /** @var Cache $cache  */
-        $cache = Core::make($this->cacheName);
-        $item = $cache->getItem('zend/'.$normalizedKey);
-
-        if ($result = $item->set($value, $this->cacheLifetime)) {
-            $item->save();
+        $cache = $this->getCache();
+        if ($cache === null) {
```

Comment:
With this pattern, I'd just do:
```php
if ($cache = $this->getCache()) {
    $item = ...
    return true;
}

return false;
```

Also seeing the 'zend/'.$normalizedKey) here makes me thing this might benefit from a protected method like `$this->getCacheKey($normalizedKey)`